apps/freeablo/faworld/behaviour.cpp

```diff
@@ -34,8 +34,10 @@ namespace FAWorld
         return nearest;
     }
 
-    void BasicMonsterBehaviour::update(size_t ticksPassed)
+    void BasicMonsterBehaviour::update()
     {
+        Tick ticksPassed = World::get()->getCurrentTick();
```

Comment:
This might get pretty annoying when we will try to write tests.  Especially these behaviour classes look like good candidates for testing because they are pretty stateless (or at least they handle the state internally).